### PR TITLE
Rename hard-stop to critical per the 1.0 specification

### PR DESCRIPTION
**Pull request by mgrowette**
updated `hard-stop` indicator to `critical` per the CDS Hooks Spec

### CODE REVIEW

**Review by kolkheang — APPROVED** [review]
@migrow Thanks for the contribution! Can you add a link to the spec or ballot for this change?

**Comment by mgrowette** [comment]
Sure thing!

'Urgency/importance of what this card conveys. Allowed values, in order of increasing urgency, are: info, warning, critical. The EHR MAY use this field to help make UI display decisions such as sort order or coloring.'
https://cds-hooks.org/specification/1.0/#card-attributes

File: src/components/CardList/card-list.jsx [hunk 1]
```diff
@@ -175,14 +175,14 @@ export class CardList extends Component {
     const indicators = {
       info: 0,
       warning: 1,
-      'hard-stop': 2,
+      'critical': 2,
```
Comment (kpshek) [inline src/components/CardList/card-list.jsx]: Can you remove the single quotes around `'critical'`? They were needed when this was `hard-stop` (due to the dash), but now we can remove them which puts it inline with the other indicator values.

File: src/components/CardList/card-list.jsx [hunk 2]
```diff
@@ -175,14 +175,14 @@ export class CardList extends Component {
...
       error: 3,
     };
 
     const summaryColors = {
       info: '#0079be',
       warning: '#ffae42',
-      'hard-stop': '#c00',
+      'critical': '#c00',
```
Comment (kpshek) [inline src/components/CardList/card-list.jsx]: Same comment above regarding the single quotes

**Review by kpshek — CHANGES_REQUESTED** [review]
Thanks so much for this contribution @migrow!

One change you missed was in `card-list.css`.  There is an `.alert-hard-stop` CSS class that needs to be renamed to `.alert-critical`.  The following line in `card-list.jsx` applies the CSS class based upon the card's indicator:

```js
const classes = cx(styles['decision-card'], styles.alert, styles[`alert-${card.indicator}`]);
```

**Review by kpshek — APPROVED** [review]
Thanks @migrow!

**Pull request merged**